.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-09be9870437684ba2da6741af3eb10126c04aede
+8a78bec5dbb43c1047c30abffe89ac622ac7911b
```

Comment:
is this intended?

Reply:
no, the XLA change was landed to master so I can update this to XLA tip

Reply:
oh yeah but I do need an XLA pin update haha